openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py

```diff
@@ -122,9 +121,14 @@ def horiz_comp_to_geom_mean(self, ctx, imt):
                     conv_sigma = (C[2] + (C[3]-C[2]) *
                                   np.log10(T/0.15)/np.log10(0.8/0.15))
                 rstd = C[4]
+    elif comp in ['GEOMETRIC_MEAN']:
+        conv_median = 1
+        conv_sigma = 1
     else:
+        conv_median = 1
+        conv_sigma = 1
         msg = f'Conversion not applicable for {comp}'
-        raise ValueError(msg)
+        warnings.warn(msg, UserWarning)
 
     # Original total STD
     total_stddev = getattr(self, const.StdDev.TOTAL)
```

Comment:
line 137 fails when the `elif` or `else` is used, since rstd has not been defined. I think it should be 1 in these cases (but also see my comment on line 124)

Reply:
yes rstd should be = 1